chapters/angulardart/index.md

```diff
@@ -257,7 +254,8 @@ Connectors are the pieces of code of the architecture that link the elements, or
 #### External entities
 AngularDart has "other systems, software programs, hardware devices or other entities with which the system interacts" (Rozanski and Woods, p. 272), or external entities. These can be extracted from the Context view image below.
 
-![Context view with logos](images/Context_view_with_logos.png "Context view with logos")
+![](images/Context_view_with_logos.png)
+_Context view with logos_
```

Comment:
Might be preferably to have these both below and as alt text for accessibility for screen readers and crawlers